### ISSUE TITLE
support set "forever" on valid_before field of the certificate

### ISSUE THREAD

**Issue by gzm55** [scheiblingco/sshkey-tools#15]
when sign by `ssh-keygen`, in valid_before field, the "forever" is translated to `2**64-1`. This behavior is not supported by `sshkey-tools` now.

**Comment by scheibling** [comment]
I'll put it in the next update (EOW) along with restoring the functionality for choosing the RSA Algo. Thanks for the feedback!

**Comment by scheibling** [comment]
Fixed in #17 , you can now specify a string like "2w", "1d4h" or similar (according to the specification in [pytimeparse2](https://github.com/onegreyonewhite/pytimeparse2), including "forever" which will set the expiration time to MAX_INT64.

Also updated the docs in #18 

It'll be included in upcoming release 0.9.2 on pypi

**Comment by gzm55** [comment]
@scheibling `MAX_INT64` is `2**64`, which is too large for 64 bit integer.  The stack is

```python
    return pack(">Q", value)
           ^^^^^^^^^^^^^^^^^
struct.error: int too large to convert
```

**Comment by scheibling** [comment]
Saw this earlier but it seems it managed to slip my mind.. I'll fix it, thanks for checking. Have somewhere on my to-do list to add some more tests as well to catch things like this.

**Comment by scheibling** [comment]
Resolved through #17 and bugs in #20 #21
